DataProducts/DPP/Energy/Battery/LastChargeEntry_v0.1.json

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
For simplicity for now let's leave the Energy out of the path now for simplicity. We do not have that many product categories coming so let's leave the upper level ontology out for now.

So I would suggest these for the paths for the first DPP products

DigitalProductPassport/Battery/DataSheet
DigitalProductPassport/Battery/Manufacturing/EnvironmentalFootprint
DigitalProductPassport/Battery/UsageHistory